testing/internal/e2e/boundary/bucket.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package boundary
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/boundary/api/storagebuckets"
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+// CreateNewStorageBucket uses boundary CLI to create new Storage Bucket in the us-east-1 region
+func CreateNewStorageBucket(t testing.TB, ctx context.Context, orgId, bucketName string) string {
+	output := e2e.RunCommand(ctx, "boundary",
+		e2e.WithArgs(
+			"storage-buckets", "create",
```

Comment:
If we're reusing the bucket for multiple tests, we may want to specify a unique recordings directory when creating the bucket. I think it's the `bucket-prefix` option